byway/src/main/webapp/destinations.js

```diff
@@ -133,17 +129,52 @@ function addMarker(searchBox,map){
 * fetches start location and destinations from DestinationsServlet and adds to DOM
 */
 function updateLocations(locationData){
-    document.getElementById('start-location').innerText = "Start Location :" + locationData.start;
     const container = document.getElementById('destinations-container');
-    container.innerText = "Destinations:";
+    container.innerText = "";
     let destinationArray= locationData.destinations;
     destinationArray.forEach((destination) => {
-      let destinationToAdd = document.createElement('p');
-      destinationToAdd.innerText = destination;
-      container.appendChild(destinationToAdd);
+      const request = {
+        query: destination,
+        fields: ["name", "photos", "formatted_address", "rating", "business_status"]
+      };
+      let service = new google.maps.places.PlacesService(map);
+      service.findPlaceFromQuery(request, (results, status) => {
+        if (status === google.maps.places.PlacesServiceStatus.OK) {
+          addLocationtoDOM(results, container);
+        }
+        else{
+          let destinationToAdd = document.createElement('p');
+          destinationToAdd.className = 'location';
+          destinationToAdd.innerText = destination;
+          container.appendChild(destinationToAdd);
+        }
+      });  
     }) 
 }
 
+/** 
+* adds Users Input Destinations to DOM with image and address
+*/
+function addLocationtoDOM(results,container){
+  let destinationToAdd = document.createElement('p');
```

Comment:
I'd recommend organizing this small self contained section into `div`s with a shared class to make it more organized on the page. Also, this allows you to target this elements in your css without having to specify as many classes. E.g. instead of having a `destiantion-photo' class, you could have 

```
.location img {
  ...
}
```

Reply:
I fixed this to add the img and destination-info the the class location